lib/matplotlib/patches.py

```diff
@@ -338,6 +340,14 @@ def get_facecolor(self):
         """Return the face color."""
         return self._facecolor
 
+    def get_hatchcolor(self):
+        """Return the hatch color."""
+        if self._hatch_color == 'edge':
+            if self._edgecolor[3] == 0:
```

Comment:
```suggestion
            if self._edgecolor[3] == 0:  # fully transparent
```
I had to pause and think about what's happening here. The small comment should make that easier to understand.